.github/workflows/build_docs.yml

```diff
@@ -67,14 +67,11 @@ jobs:
         echo "::endgroup::"
         echo "::group::Install TorchAudio"
         conda install --quiet --yes cmake>=3.18.0 ninja
+        conda install "ffmpeg<5" -c conda-forge
```

Comment:
Why is this required?

Reply:
Per [examples/README.md](https://github.com/pytorch/audio/pull/4086/files#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9), we cannot expect that codes under examples work. Apparently these work when ffmpeg is installed properly for torchcodec. 

Do we want to depend on ffmpeg for building torchaudio docs?
^ @NicolasHug

Reply:
We can updated conda for docs build in a separate PR. What do you think?

Reply:
Sounds good. I'll copy the approach to this PR